Documentation/network/bgp-control-plane/bgp-control-plane.rst

```diff
@@ -78,4 +82,5 @@ Troubleshooting and Operation Guide
     :glob:
 
     bgp-control-plane-troubleshooting
-    bgp-control-plane-operation
\ No newline at end of file
+    bgp-control-plane-operation
+
```

Comment:
Sorry I wasn't clear. I meant that your editor added a new line character at the end of the line, which was why we'd observe a diff here. I did not mean you should add an additional new empty line.

Reply:
Oh alright!